docs/model-dev-guide/dtrain/dtrain-introduction.rst

```diff
@@ -8,56 +8,56 @@
  How Determined Distributed Training Works
 *******************************************
 
-Distributed training in Determined utilizes data-parallelism. Data-parallelism for deep-learning
-consists of a set of workers, where each worker is assigned to a unique compute accelerator such as
-a GPU or a TPU. Each worker maintains a copy of the model parameters (weights that are being
-trained), which is synchronized across all the workers at the start of training.
+Determined employs data parallelism in its approach to distributed training. Data parallelism for
+deep learning consists of a set of workers, where each worker is assigned to a unique compute
+accelerator such as a GPU or a TPU. Each worker maintains a copy of the model parameters (weights
+that are being trained), which is synchronized across all the workers at the start of training.
 
 After initialization is completed, distributed training in Determined follows a loop where:
 
 #. Every worker performs a forward and backward pass on a unique mini-batch of data.
-#. As the result of the backward pass, every worker generates a set of updates to the model
-   parameters based on the data it processed.
-#. The workers communicate their updates to each other, so that all the workers see all the updates
+#. As a result of the backward pass, every worker generates a set of updates to the model parameters
+   based on the data it processed.
+#. The workers communicate their updates to each other so that all the workers see all the updates
    made during that batch.
 #. Every worker averages the updates by the number of workers.
 #. Every worker applies the updates to its copy of the model parameters, resulting in all the
    workers having identical solution states.
 #. Return to the first step.
 
-***********************************************
- Reduce Computation and Communication Overhead
-***********************************************
+*************************************************
+ Reducing Computation and Communication Overhead
```

Comment:
should this section belong in `Debug Performance Bottlenecks`? it's a bit confusing to me how this document is organized. feel like i have to read the whole thing, each section doesn't necessarily cleanly encapsulate its info.

Reply:
yes this section belongs with debugging performance bottlenecks.

Reply:
resolved, added a new top level section, "Performance Optimization" and underneath you'll see the two sections: "Reducing Computation and Communication Overhead" and "Debugging Performance Bottlenecks"